html/donate/en.html

```diff
@@ -0,0 +1,98 @@
+
+<!doctype html>
+<html class="no-js" lang="fr">
+<head>
+<meta charset="utf-8">
+
+<title>Donate to Open Food Facts</title>
+
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+<meta property="fb:app_id" content="219331381518041">
+<meta property="og:type" content="food">
+<meta property="og:title" content="">
+<meta property="og:url" content="https://world.openfoodfacts.org/donate-to-openfoodfacts">
+
+<meta property="og:image" content="https://static.openfoodfacts.org/images/misc/openfoodfacts-logo-en-356.png">
+<meta property="og:description" content="A collaborative, free and open database of ingredients, nutrition facts and information on food products from around the world">
+<link rel="manifest" href="/cgi/manifest.pl">
+<link rel="mask-icon" href="https://static.openfoodfacts.org/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
+<link rel="shortcut icon" href="https://static.openfoodfacts.org/images/favicon/favicon.ico">
+<meta name="msapplication-TileColor" content="#da532c">
+<meta name="msapplication-TileImage" content="https://static.openfoodfacts.org/images/favicon/mstile-144x144.png">
+<meta name="msapplication-config" content="https://static.openfoodfacts.org/images/favicon/browserconfig.xml">
+<meta name="_globalsign-domain-verification" content="2ku73dDL0bAPTj_s1aylm6vxvrBZFK59SfbH_RdUya" />
+<meta name="apple-itunes-app" content="app-id=588797948">
+<meta name="flattr:id" content="dw637l">
+
+<link rel="stylesheet" href="https://static.openfoodfacts.org/css/dist/app.css?v=1562940554">
+<link rel="stylesheet" href="https://static.openfoodfacts.org/css/dist/jqueryui/themes/base/jquery-ui.min.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.3/css/select2.min.css" integrity="sha384-HIipfSYbpCkh5/1V87AWAeR5SUrNiewznrUrtNz1ux4uneLhsAKzv/0FnMbj3m6g" crossorigin="anonymous">
+<link rel="search" href="https://fr.openfoodfacts.org/cgi/opensearch.pl" type="application/opensearchdescription+xml" title="Open Food Facts">
```

Comment:
`fr.openfoodfacts.org` is the wrong subdomain for the English file

Reply:
See #2538